### ISSUE TITLE
Run as a service

### ISSUE THREAD

**Issue by tomvanrossom** [jhipster/jhipster-registry#36]
I would like to run the jhipster registry as a service [link](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/html/deployment-install.html#deployment-initd-service)

It is working fine for the jhipster gateway.

Am I missing something, or is it not implemeted (yet)?

Thanks

**Comment by PierreBesson** [comment]
@tomvanrossom I have no experience with running an app as an initd service but the JHipster Registry is a regular Spring Boot app so I don't understand why it doesn't behave the same way. 

The only difference with regular JHipster app is that it is using `spring-cloud-starter-parent` instead of `spring-boot-starter-parent`. Maybe this could be the reason why it doesn't work.

**Comment by tomvanrossom** [comment]
Thanks @PierreBesson for your reply. 

I've found the solution: the registry is runnable as a service, but only in the 'prod' profile.

I've just added a build section in the 'dev' profile:

```xml
<profile>
    <id>dev</id>
    <activation>
        <activeByDefault>true</activeByDefault>
    </activation>
    <properties>
        <!-- log configuration -->
        <logback.loglevel>DEBUG</logback.loglevel>
    </properties>
    <build>
        <plugins>
            <plugin>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-maven-plugin</artifactId>
                <configuration>
                    <executable>true</executable>
                    <arguments>
                        <argument>--spring.profiles.active=dev</argument>
                    </arguments>
                </configuration>
            </plugin>
        </plugins>
    </build>
</profile>
```

And now I can start it as a service.

**Comment by PierreBesson** [comment]
I really don't think we should encourage the registry to be run in dev profile. @jdubois do you think we should add those lines to the pom to enable this specific use case ?

**Comment by jdubois** [comment]
There is no need to run it in "dev" profile, besides this profile doesn't do anything specific today

**Comment by jd-carroll** [comment]
This can actually be closed as it was fixed with #38.  I didn't see this when I opened the issue #37 (which is actually a duplicate of this).

A reason for running a dev profile is to allow deployment into a non-prod non-localhost environment where the config repository is separate from the prod config repository.

**Comment by jdubois** [comment]
I'm closing this as per the previous comment - I'm not totally sure if this is correct, so please add comments if you disagree!!
